### PR TITLE
plugin: add support for korean ffxiv v5.01

### PR DESCRIPTION
**Pull request by KattTails**
Hiya, 

Quick addition of an extra `FFXIVProcessKo` to support the recent Korean release. It's untested as I don't have access to a Korean build but I'm hoping the offsets are unchanged.

### CODE REVIEW

File: CactbotOverlay/CactbotEventSource.cs [hunk 1]
```diff
@@ -189,10 +189,12 @@ public override void SaveConfig(IPluginConfig config)
         LogInfo("Language: {0}", language_);
       }
 
-      // Temporarily target cn if plugin is old v2.0.4.0
-      if (language_ == "cn" || ffxiv.ToString() == "2.0.4.0") {
+      if (language_ == "cn") {
```
Comment (KattTails) [inline CactbotOverlay/CactbotEventSource.cs]: I removed the hard check on the version string since new releases of the FFXIV Plugin have cn support.

Comment (quisquous) [reply CactbotOverlay/CactbotEventSource.cs]: But does the ffxiv plugin that the cn and ko players are using have that yet? I thought they were all using an older version.

Comment (KattTails) [reply CactbotOverlay/CactbotEventSource.cs]: Oh, I was falsely assuming they were backwards-compatible without really thinking it through. I'll re-add that line for cn for now.

I wonder if there's anyone who knows which version of the plugin the Korean players are using?

Comment (Bluefissure) [reply CactbotOverlay/CactbotEventSource.cs]: @devunt Hi, can you help this about the Korean act plugin version?

Comment (Bluefissure) [reply CactbotOverlay/CactbotEventSource.cs]: @devunt Hi~ Can you help with this Korean act plugin version problem?

Comment (quisquous) [reply CactbotOverlay/CactbotEventSource.cs]: I talked to @hibiyasleep who said that they used the 2.0.4.0 version directly (with no translations).  I'm not sure that there's any way to differentiate the language then until the ffxiv plugin is new enough to include cn and ko language options.

So, maybe this is good enough for now.

**Pull request merged**